src/Widget/RegionService.php

```diff
@@ -56,6 +56,10 @@ public function getAutocompletResults($searchString, $language = 'nl')
                 if (strpos(strtolower($translatedRegion), '(' . $searchString) !== false) {
                     $matches[$region->key] = $translatedRegion;
                 }
+                // This is done to add informal municipality names without prefixes like Saint
```

Comment:
Maybe we can store the lower string version of `$translatedRegion` inside a variable?